vtex-apps/login
# Feature/customization classes

#### What is the purpose of this pull request?

Add new CSS customization classes

#### What problem is this solving?

This creates separate CSS customization classes for each login option button. It also creates separate classes for three different forms that shared the same class.

#### How should this be manually tested?

Go into
https://rafaprtest3--storecomponents.myvtex.com/
and open the login app. You can inspect its components to see the new classes, for example, Facebook's 
button:

![image](https://user-images.githubusercontent.com/22064061/70460781-e2752a00-1a95-11ea-9183-d56c7360d542.png)

![image](https://user-images.githubusercontent.com/22064061/70460877-181a1300-1a96-11ea-9678-88625008bc6f.png)

#### Types of changes
